Tasks/AzureTestPlanV0/task.json

```diff
@@ -125,6 +125,18 @@
         "defaultValue": false,
         "required": false,
         "helpMarkDown": "Fail the task if no result files are found."
+    },
+    {
+      "name": "mavenPOMFile",
```

Comment:
Just for my understanding - Is this an additional input customer needs to provide?

Reply:
Yes. This option is parameter is necessary for Maven execution

Reply:
Do we have an option to only display this when java is selected?

Reply:
yes, we should ask for this option only when java maven is selected, as it is required in that case only. Not required for python or gradle.

We can achieve this similar to this:

"visibleRule": "testSelector = testAssemblies"

Reply:
It is already added.

Reply:
I have removed the input for POM file since it was causing errors for multi-select inputs of testLanguage